assemblies/monitoring-data-drift.adoc

```diff
@@ -9,11 +9,22 @@ ifdef::context[:parent-context: {context}]
 
 [role='_abstract']
 
-As a data scientist, you might want to monitor your deployed models for data drift. Data drift refers to changes in the distribution or properties of incoming data that differ significantly from the data on which the model was originally trained. Detecting data drift helps ensure that your models continue to perform as expected and that they remain accurate and reliable.
+Data drift refers to changes that occur in the distribution of incoming data that differ significantly from the data on which the model was originally trained. This distributional shift can cause model performance to become unreliable, as machine learning models rely heavily on the patterns in their training data. .
```

Comment:
```suggestion
Data drift refers to changes that occur in the distribution of incoming data that differ significantly from the data on which the model was originally trained. This distributional shift can cause model performance to become unreliable, because machine learning models rely heavily on the patterns in their training data. .
```